wiki/Article_styling_criteria/Writing/th.md

```diff
@@ -0,0 +1,336 @@
+
+```markdown
+Dean Herbert (also known as peppy) created osu! back in 2007.
+```
+
+### เครื่องหมายการค้า
+
+*สำหรับกฎเกี่ยวกับ osu! โปรดดู: [osu!](#osu!)*
+
+เครื่องหมายการค้าต่อไปนี้ต้องสะกดดังนี้ (ตัวพิมพ์ต้องตรงกัน):
+
+- `Discord`
+- `Facebook`
+- `GitHub`
+- `Google`
+- `Reddit`
+- `Skype`
+- `Twitch`
+- `Twitter`
+- `YouTube`
+
+เครื่องหมายการค้าต้องไม่ตามด้วยเครื่องหมายการค้าหรือเครื่องหมายการค้าจดทะเบียน
+
+## วันและเวลา
+
+### การจัดรูปแบบวันที่
+
+รูปแบบที่พบใน [วิกิพีเดีย:คู่มือรูปแบบ/วันที่และตัวเลข § รูปแบบ](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Formats) เป็นรูปแบบวันที่ที่ยอมรับทั้งหมด การจัดรูปแบบวันที่ควรสอดคล้องกันตลอดทั้งบทความ
+
+รูปแบบวันที่ `ปปปป-ดด-วว` ควรใช้ในตารางเท่านั้น
+
+### การจัดรูปแบบเวลา
+
+การจัดรูปแบบเวลาต้องทำโดยใช้รูปแบบต่อไปนี้:
+
+```markdown
+ชั่วโมงเลข 2 ตัว:นาทีเลข 2 ตัว เขตเวลา
+```
+
+เวลาเขียนในรูปแบบ 24 ชั่วโมง และตัวเลขเดี่ยวใดๆ จะต้องนำหน้าด้วยศูนย์ เขตเวลาจะต้องเขียนทันทีหลังจากเวลานั้น และควรอยู่ใน UTC+0 สำหรับเหตุการณ์ทั่วโลก หรือในเขตเวลาของกิจกรรมสำหรับกิจกรรมที่มีขนาดเล็กกว่า ใช้ `UTC` แทน `GMT`
+
+ตัวอย่างที่ไม่ดี:
+
+```markdown
+3:30 PM UTC
+22:30 (UTC+7)
+11:30
+```
+
+ตัวอย่างที่ดี:
+
+```markdown
+15:30 UTC
+22:30 UTC+7
+11:30 UTC-4
+```
+
+`UTC` (ไม่มีออฟเซ็ต) ถูกบอกเป็นนัยว่าเป็น `UTC+0` อาจใช้อย่างใดอย่างหนึ่ง แต่การใช้งานต้องสอดคล้องกัน UTC ที่มีออฟเซ็ต 0 อย่างชัดเจนต้องใช้สัญลักษณ์บวก (`+`)
+
+### การจัดรูปแบบวันที่และเวลา
+
+เมื่อใช้ทั้งวันที่และเวลาร่วมกัน ต้องเขียนวันที่ก่อน ตามด้วยเวลา เวลาต้องอยู่ระหว่างวงเล็บ (`(` และ `)`)
+
+ตัวอย่างที่ไม่ดี:
+
+```markdown
+October 25, 2016 at 11:45 UTC
+October 25, 2016 11:45 UTC
+```
+
+ตัวอย่างที่ดี:
+
+```markdown
+October 25, 2016 (11:45 UTC)
+```
+
+## โหมดเกม
+
+โหมดเกมจะต้องเขียนดังนี้:
+
+- `osu!`
+- `osu!taiko`
+- `osu!catch`
+- `osu!mania`
+
+การอ้างถึงชื่อโหมดเกมแบบเก่า (เช่น `Catch the Beat`, `Taiko` และ `Mania`) อาจทำได้หากพูดถึงชื่อเดิมของโหมดเกมดังกล่าว
+
+## osu!
+
+ชื่อของเกม osu! ต้องไม่เป็นตัวพิมพ์ใหญ่หรือตัวเอียง โอสุ! ตราสินค้าอย่างเป็นทางการจะต้องไม่ใช้ช่องว่างใด ๆ ตัวอย่างได้แก่
+
+- `osu!academy`
+- `osu!api`
+- `osu!catch`
+- `osu!direct`
+- `osu!keyboard`
+- `osu!mania`
+- `osu!store`
+- `osu!stream`
+- `osu!supporter`
+- `osu!tablet`
+- `osu!taiko`
+- `osu!talk`
+- `osu!tourney`
+
+ชื่อผู้ใช้ที่มี `osu!` ซึ่งเป็นส่วนหนึ่งของชื่อหัวข้อจะต้องเป็นตัวพิมพ์ใหญ่ ตัวอย่างได้แก่
+
+- `osu! Alumni`
+- `osu! Champion`
+
+สำหรับเงื่อนไขอื่นๆ ทั้งหมด osu! ต้องถือว่าเป็น[คำนามที่มีคุณสมบัติ](https://en.wikipedia.org/wiki/Noun_adjunct) นี่หมายถึงการเพิ่มช่องว่างระหว่าง osu! และคำนามที่ปรับเปลี่ยน ตัวอย่างได้แก่
+
+- `osu! tournaments`
+- `osu! community`
+- `osu! chat`
+- `osu! client`
+- `osu! wiki`
+
+---
+
+`osu!` ไม่ควรจบประโยค หากใช่ ให้ใช้จุดเต็ม (`.`) หรือเครื่องหมายคำถามเพื่อรับ `osu!.` หรือ `osu!?`
+
+`osu!` ต้องไม่ตามด้วยเครื่องหมายอัศเจรีย์ (เช่น `osu!!`)   ประโยคจะต้องถูกเขียนใหม่เพื่อให้แน่ใจว่าสิ่งนี้จะไม่เกิดขึ้น
+
+## คำศัพท์
+
+คำเหล่านี้ต้องสะกดดังนี้ (การวรรคต้องตรงกัน):
```

Comment:
this is also useless, unless you use words from the below list as-is even in translations. want to replace them with local equivalents?

typically, sections like these are used for words or collocations that may be misspelled in _your_ language, or have several spelling variations, and you need to pick one for consistency

Reply:
idk bout this though

Reply:
@Trigonoculus what should I do with this?
most of these words aren't in dictionary mostly

Reply:
Some of these are better left untranslated (at least in my opinion). In our community, there are no "proper" translations to some of these. For example, Thai people called "approach circle" a "approach circle". However, some of these can definitely be transliterated to Thai (Well except approach circle cause that would be something like "อะโพรช เซอร์เคิล" which is probably harder to read than English).

Reply:
resolved
thanks @C0NS03L <3